app/assets/stylesheets/responsive/_request_style.scss

```diff
@@ -334,6 +334,17 @@ div.comment_in_request {
   background-color: mix(#fff, $color-delivery-unknown, 85%);
 }
 
+.correspondence__suggestion {
+  background-color: #eee;
```

Comment:
> The message has a pale grey background by default in the core theme. We should probably override that to something a bit more colourful in the WDTK theme – I guess that should be done in a pull request on whatdotheyknow-theme?

I think we can set a sensible default colour here, since it's a core part of the interface. I don't want to have to remember to style this on _every_ theme we implement it on. I was thinking a pale yellow?

Reply:
Fixed in 679d190